jupyter_client/tests/test_multikernelmanager.py

```diff
@@ -2,13 +2,14 @@
 
 import asyncio
 import threading
+import uuid
 import multiprocessing as mp
 
 from subprocess import PIPE
 from unittest import TestCase
 from tornado.testing import AsyncTestCase, gen_test
-
 from traitlets.config.loader import Config
+from ipython_genutils.py3compat import unicode_type
```

Comment:
Since we're python 3 only now I don't think you need to import this and can just use `str`

Reply:
Right, and it's also used in `multikernelmanager.py`. It could be good to make the change there also.

Reply:
Going to merge for now, we can do a python 3 compat cleanup later on